homeassistant/helpers/script.py

```diff
@@ -99,11 +99,17 @@ def async_script_delay(now):
                 delay = action[CONF_DELAY]
 
                 try:
+                    import collections
```

Comment:
We import library on top like PEB8

Reply:
Yes, you're right. Realized I mixed this up with the policy on importing dependencies in components being function level to make sure they get installed. On second thought, we actually don't need collections, as simply validating with a dict should be sufficient.